test/test_opam_cmd.ml

```diff
@@ -40,3 +40,55 @@ let test_tag_from_archive =
     make_test ~archive:"https://other.domain.com/some/path/file-v1.2.3.tbz"
       ~expected:(Some "v1.2.3") ()
   ]
+
+let test_classify_package =
+  let make_test ~name ~package ?(dev_repo = "dummy-dev-repo") ?archive ?(pins = []) ~expected () =
```

Comment:
Why not taking `~name` and `?version` arguments instead of `~package` ?

Reply:
This is just to stick as much as possible to the signature of the function understand and avoid adding logic to the test as much as possible. It makes it easier to guess what the test will do i.e. just pass the arguments along to the function.

I agree in this case it seems acceptable but I'm not sure we'd gain much.